### ISSUE TITLE
Missing HTML anchors for Python 3 docset

### ISSUE THREAD

**Issue by IngoMeyer441** [sunaku/dasht#58]
Thanks for this nice project, doing as much as posible in the terminal is always awesome! :smiley: 

I tried `dasht` with the Python 3 docset. If I search for `sys.argv` for example, `dasht` finds `sys.argv` but the link behind it navigates to the top of the `sys` documentation page. Zeal jumps to the `sys.argv` section instead. I think there is a missing `#sys.argv` at the end of the found documentation URL. It this behavior docset dependent?

**Comment by sunaku** [comment]
Yes, this is a known issue (see #16) and pull requests are welcome. :pray:  Long ago, I installed all available docsets and manually examined each of them to learn their individual style of URI fragments ("HTML anchors").  Some had them and others didn't.

**Comment by IngoMeyer441** [comment]
Ah I see :disappointed:. Hmm, but how does Zeal handle this? Maybe dasht can be improved this way.

**Comment by IngoMeyer441** [comment]
I think the relevant Zeal code is in this file: <https://github.com/zealdocs/zeal/blob/74ca9bfef09ad5216b42fc1cc785734351e9726c/src/libs/registry/docset.cpp>.

**Comment by sunaku** [comment]
Hi Ingo, I didn't look into the Zeal codebase link you provided yet.  Instead, I tried installing the "python_3" docset on my end and attempted your search for `sys.argv` but I couldn't reproduce the problem! :bulb:  On my system, both the terminal based `dasht` search as well as the web based `dasht-server` search properly took me to the exact anchor in the HTML document.

Is your Python 3 docset installation up to date?  Mine has the following timestamp and checksum.  Please check.

```sh
$ cd ~/.local/share/dasht/docsets/
$ sha256sum Python_3.tgz
4c69f15063840872d855a3a1c7314f927c57c8fe1f48cfbe02fe24ab84bf0ad3 Python_3.tgz
$ date -Is -r Python_3.tgz
2022-12-08T03:29:22-08:00
```

**Comment by IngoMeyer441** [comment]
Hi @sunaku,
sorry for my late reply. I set `DASHT_DOCSETS_DIR` to my Zeal docset directory to avoid duplicates. There are no tgz-files inside I could use for a checksum. But I tried to unset `DASHT_DOCSETS_DIR` again and installed the Python 3 docset with

```bash
dasht-docsets-install Python_3
```

When using the `dasht` docset, jumping to the correct doc location works. :smile: Sorry, I was not aware that the docsets of Dasht and Zeal differ. Is there no compatibility between the two?
